geemusic/intents/selection.py

```diff
@@ -97,106 +145,78 @@ def play_song(song_name, artist_name):
+#   #                              artist['name'], artist_id=artist['artistId'])
+#    # _artist = int(artist['name'])
+#    # _artistId = int(artist['artistId'])
+#    station_id = api.get_station("%s Radio" % artist[0], artistId=artist['artist_id'])
+#    # TODO: Handle track duplicates (this may be possible using session ids)
+#    tracks = api.get_station_tracks(station_id)
 
-    if song is False:
-        return statement("Sorry, I couldn't find that song")
+#    first_song_id = queue.reset(tracks)
 
-    station_id = api.get_station("%s Radio" %
-                                 song['title'], track_id=song['storeId'], artist_id=artist['artistId'], album_id=album['albumId'])
-    tracks = api.get_station_tracks(station_id)
-
-    first_song_id = queue.reset(tracks)
-    stream_url = api.get_stream_url(first_song_id)
-
-    thumbnail = api.get_thumbnail(queue.current_track()['albumArtRef'][0]['url'])
-    speech_text = "Playing %s by %s" % (song['title'], song['artist'])
-    return audio(speech_text).play(stream_url) \
-        .standard_card(title=speech_text,
-                       text='',
-                       small_image_url=thumbnail,
-                       large_image_url=thumbnail)
+#    # Get a streaming URL for the top song
+#    stream_url = api.get_stream_url(first_song_id)
 
+#    thumbnail = api.get_thumbnail(artist)
+#    speech_text = "Playing %s radio" % artist['name']
+#    return audio(speech_text).play(stream_url) \
+#        .standard_card(title=speech_text,
+#                       text='',
+#                       small_image_url=thumbnail,
+#                       large_image_url=thumbnail)
 
 @ask.intent("GeeMusicPlayArtistRadioIntent")
-def play_artist_radio(artist_name):
+def play_artist_radio(name):
+	if str(environ['USE_LIBRARY_FIRST']) is True:
+		# return statement("Sorry, artist radios aren't currently supported at the moment.")
+		return None
     # Fetch the artist
-    artist = api.get_artist(artist_name)
+	artist = api.get_artist(name)
 
-    if artist is False:
-        return statement("Sorry, I couldn't find that artist")
+	if artist is False:
+		return statement("Sorry, I couldn't find that artist")
 
-    station_id = api.get_station("%s Radio" %
-                                 artist['name'], artist_id=artist['artistId'])
-    # TODO: Handle track duplicates (this may be possible using session ids)
-    tracks = api.get_station_tracks(station_id)
 
-    first_song_id = queue.reset(tracks)
 
-    # Get a streaming URL for the top song
-    stream_url = api.get_stream_url(first_song_id)
+    # station_id = api.get_station("%s Radio" % artist['name'], artist_id=artist['artistId'])
+	station_id = api.get_station("%s Radio" % artist['name'], artist_id=artist['artistId'])
 
-    thumbnail = api.get_thumbnail(artist['artistArtRef'])
-    speech_text = "Playing %s radio" % artist['name']
-    return audio(speech_text).play(stream_url) \
+    #if str(environ['USE_LIBRARY_FIRST']) is True:
```

Comment:
Why commenting these?

Reply:
This was an attempt to get radios to work for free accounts. I was getting close to deducing what was causing radios to return errors on free accounts, which was due to the gmusicapi package not being up to date (the package has been updated so that radios can work with free Google Play Music accounts). I left it in there as notes for someone else to look at it, and maybe extend upon my work.